download.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: "Download Terasology Launcher"
+header-img: "/img/ss_3.jpg"
+---
+
+<div class="row ">
+  <div class="col s6 l2 center">
+    <h4>Select OS:</h4>
+  </div>
+  <div class="col s6 l7 center">
+    <select style="display: block;" id="OS">
+      <option>macOs (64-Bit)</option>
```

Comment:
MacOS is only 64-bit.

```suggestion
      <option>MacOS</option>
```